frontend/packages/console-dynamic-plugin-sdk/src/extensions/cluster-settings.ts

```diff
@@ -9,9 +9,12 @@ export type ClusterGlobalConfig = ExtensionDeclaration<
     name: string;
     /** The model which refers to a cluster config resource. */
     model: {
-      group: string;
-      version: string;
+      apiVersion: string;
+      apiGroup: string;
       kind: string;
+      plural?: string;
+      namespaced?: boolean;
+      crd?: boolean;
```

Comment:
As discussed in slack, keep the current api but transform the model to appropriate data structure for the util.

Reply:
have updated it